packages/manager/modules/core/src/redirection/redirection.service.js

```diff
@@ -1,14 +1,33 @@
+import forOwn from 'lodash/forOwn';
 import get from 'lodash/get';
 import constants from './redirection.constants';
 
 export default class RedirectionService {
   /* @ngInject */
-  constructor(coreConfig) {
+  constructor(coreConfig, SessionService) {
     this.coreConfig = coreConfig;
+    this.SessionService = SessionService;
   }
 
-  getURL(id) {
+  getURL(id, params = {}) {
     const regionConstants = constants[this.coreConfig.getRegion()];
-    return get(regionConstants, id);
+    let url = get(regionConstants, id);
+    if (url) {
+      const keyValidator = /^[\w-]+$/;
+      forOwn(params, (value, key) => {
+        if (keyValidator.test(key)) {
+          url = url.replace(`:${key}`, encodeURIComponent(value));
+        }
+      });
+    }
+    return url;
+  }
+
+  getGuideURL(id) {
+    return this.SessionService.getUser().then(({ ovhSubsidiary }) => {
```

Comment:
Do you think maybe the `subsidiary` could be passed as a parameter ? Because for quite a few URLs we will have different values depending on the subs and we will have to fetch `/me` each time

Reply:
Yes i was not happy with the getGuideURL method we should have a single getURL method that handle all the cases. I'm going to refactor it